### DISCUSSION TITLE
eCommerce?

### DISCUSSION THREAD

**Discussion by AdamJel** [schultek/jaspr#306] [Q&A] [answered]
Hello, I just found the 'jaspr' and since I use Flutter for web apps, I will definitely give a go for static and CMS websites. But my question is - are there any pre-build solutions for eCommerce in jaspr? Is it somewhere down the roadmap?

Thank you 🙏🏻

**Comment by schultek** [comment, marked as answer]
Hi. I'm not aware of any pre-built solution right now. I also don't think this should come from the framework, but from the community.

**Reply by AdamJel** [reply]
I haven't found anything either. Ok, that's pitty. Hopefully something will emerge in time. I myself am not up for such a task.

As a result this means that I need different technologies for eC and not eC web projects. Oh well.

Thank you
